pkg/watch/health.go

```diff
@@ -226,7 +226,7 @@ func updatePods(current []PodWatch, reality []kp.ManifestResult, logger *logging
 
 		// if a manifest is in reality but not current a podwatch is created
 		// with that manifest and added to newCurrent
-		if missing == true {
+		if missing == true && man.Manifest.StatusPort != 0 {
```

Comment:
`== true` is redundant isn't it? I maybe should have said something before, but while we're changing this line might as well, right?